src/Stats/reducer.jsx

```diff
@@ -1,15 +1,15 @@
 import { actionType } from './actionCreator';
 
 const initialState = {
-  transactions: [],
+  transactionList: [],
 };
 
 export default (state = initialState, action) => {
   switch (action.type) {
     case actionType.ADD_TRANS:
       return {
         ...state,
-        transactions: [...state.transactions, { ...action.data }],
+        transactionList: [...state.transactionList, { ...action.data }],
```

Comment:
do not add unrelated change, if the team wants to rename it please create another branch and pr

Reply:
fixed